<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDTSampleDockWidget</class>
 <widget class="QDockWidget" name="CDTSampleDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category and Samples</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Categories</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxSamples">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Training Samples</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QFrame" name="frameSample">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Segmentation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRefresh">
              <property name="toolTip">
               <string>Refresh</string>
              </property>
              <property name="text">
               <string>Refresh</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/Icon/refresh.png</normaloff>:/Icon/refresh.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButtonEditSample">
              <property name="toolTip">
               <string>Edit Samples</string>
              </property>
              <property name="text">
               <string>Edit Samples</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/Icon/sample.png</normaloff>:/Icon/sample.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSampleRename">
              <property name="toolTip">
               <string>Rename</string>
              </property>
              <property name="text">
               <string>Rename</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/Icon/rename.png</normaloff>:/Icon/rename.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonNewSample">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Insert New Sample</string>
              </property>
              <property name="text">
               <string>Insert New Sample</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/Icon/add.png</normaloff>:/Icon/add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRemoveSelected">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Remove Selected Sample</string>
              </property>
              <property name="text">
               <string>Remove Selected Sample</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/Icon/minus.png</normaloff>:/Icon/minus.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/plus.png</normaloff>:/Icon/plus.png</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/minus.png</normaloff>:/Icon/minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionRemove_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/remove.png</normaloff>:/Icon/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/edit.png</normaloff>:/Icon/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/revert.png</normaloff>:/Icon/revert.png</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/Icon/commit.png</normaloff>:/Icon/commit.png</iconset>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="toolTip">
    <string>Submit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
